### PR TITLE
Touch improvements, Logging mask 

### PR DESCRIPTION
**Pull request by marcos-diaz**

### CODE REVIEW

File: src/logging.c [hunk 1]
```diff
@@ -23,6 +24,18 @@ bool logging_get_level() {
...
 
+void logging_set_mask(LogMask mask) {
+    logging_mask = mask;
+}
+
+bool logging_has_mask(LogMask mask) {
+    return logging_mask & mask;
+}
+
+bool logging_get_mask() {
```
Comment (przymi) [inline src/logging.c]: Returns enum, not bool

File: src/headers/touch.h [hunk 2]
```diff
@@ -3,24 +3,25 @@
 
 #pragma once
 
-// Smooting of state change for the touch reporting.
-#define CFG_TOUCH_SMOOTH 2  // Ticks.
```
Comment (przymi) [inline src/headers/touch.h:7]: Smoothing is removed? Should be in notes.

File: src/headers/touch.h [hunk 3]
```diff
@@ -3,24 +3,25 @@
...
-
-// The baseline threshold value when using dynamic.
-#define CFG_GEN0_TOUCH_DYNAMIC_MIN 3  // Microseconds
-#define CFG_GEN1_TOUCH_DYNAMIC_MIN 15 // Microseconds
+// The starting baseline threshold value when using dynamic.
```
Comment (przymi) [inline src/headers/touch.h]: dynamic => auto

**Review by przymi — CHANGES_REQUESTED** [review]

File: src/headers/touch.h (hunk 2 above)
Comment (marcos-diaz) [reply src/headers/touch.h:7]: Redefined as `CFG_TOUCH_AUTO_SMOOTH`

**Pull request merged**